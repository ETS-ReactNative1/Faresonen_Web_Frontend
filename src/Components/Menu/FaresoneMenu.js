import React from "react";
import { Menu } from "semantic-ui-react";

const FaresoneMenu = () => (
  <div className="no-print">
    <h1>Faresonen</h1>
    <Menu>
      <Menu.Item as={"a"} href="#/" name="home">
        Hjem
      </Menu.Item>
      <Menu.Item as={"a"} href="#/dangerzone" name="dangerzone">
        Faresonen
      </Menu.Item>
      <Menu.Item as={"a"} href="#/about" name="about">
        Om
      </Menu.Item>
    </Menu>
  </div>
);

export default FaresoneMenu;
