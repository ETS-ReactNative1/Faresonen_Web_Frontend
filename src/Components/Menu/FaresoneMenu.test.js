import React from "react";
import renderer from "react-test-renderer";
import FaresoneMenu from "./FaresoneMenu";

describe("<FaresoneMenu />", () => {
  it("Should render properly", () => {
    const tree = renderer.create(<FaresoneMenu />);
    expect(tree).toMatchSnapshot();
  });
});
