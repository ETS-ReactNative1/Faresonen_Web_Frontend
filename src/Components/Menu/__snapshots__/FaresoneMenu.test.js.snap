// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FaresoneMenu /> Should render properly 1`] = `
<div
  className="no-print"
>
  <div
    className="ui inverted menu"
  >
    <a
      className="item"
      href="#/"
      onClick={[Function]}
    >
      Hjem
    </a>
    <a
      className="item"
      href="#/dangerzone"
      onClick={[Function]}
    >
      Faresonen
    </a>
    <a
      className="item"
      href="#/gigsports"
      onClick={[Function]}
    >
      GIG Sports
    </a>
    <a
      className="item"
      href="#/fantasy"
      onClick={[Function]}
    >
      Fantasy PL
    </a>
    <a
      className="item"
      href="#/fantasy/stats"
      onClick={[Function]}
    >
      Statistikk
    </a>
    <a
      className="item"
      href="#/premierleague"
      onClick={[Function]}
    >
      Premier League-verkt√∏y
    </a>
    <div
      aria-busy={undefined}
      aria-disabled={undefined}
      aria-expanded={false}
      aria-multiselectable={undefined}
      className="ui item dropdown"
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      role="listbox"
      tabIndex={0}
    >
      <div
        aria-live="polite"
        className="text"
        role="alert"
      >
        Admin
      </div>
      <i
        aria-hidden="true"
        className="dropdown icon"
        onClick={[Function]}
      />
      <div
        className="menu transition"
      >
        <a
          aria-checked={undefined}
          aria-disabled={undefined}
          aria-selected={undefined}
          className="item"
          href="#/admin"
          name="admin"
          onClick={[Function]}
          role="option"
        >
          <span
            className="text"
          >
            Logg inn
          </span>
        </a>
        <a
          aria-checked={undefined}
          aria-disabled={undefined}
          aria-selected={undefined}
          className="item"
          href="#/newuser"
          name="newuser"
          onClick={[Function]}
          role="option"
        >
          <span
            className="text"
          >
            Opprett ny bruker
          </span>
        </a>
      </div>
    </div>
    <a
      className="right item"
      href="#/about"
      onClick={[Function]}
    >
      Om
    </a>
  </div>
</div>
`;
